examples/cdk/functions/common/getUuid.ts

```diff
@@ -0,0 +1,22 @@
+import { logger } from './powertools';
+import { getParameter } from '@aws-lambda-powertools/parameters/ssm';
+import { v4 } from 'uuid';
```

Comment:
maybe we can just use `randomUUID` from `node:crypto` like we do in the other endpoint